DSharpPlus.SlashCommands/EventArgs/ContextMenuReceivedEventArgs.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Emzi0767.Utilities;
+
+namespace DSharpPlus.SlashCommands.EventArgs
+{
+    /// <summary>
+    /// Represents arguments for a <see cref="SlashCommandsExtension.ContextMenuReceived"/>
+    /// </summary>
+    public sealed class ContextMenuReceivedEventArgs : AsyncEventArgs
```

Comment:
"received" isn't very apt nomenclature in this context; perhaps "ContextMenuInvoked"?

Reply:
7a5688e: Is corrected